step-plans/step-plans-yaml-parser/src/test/java/step/plans/parser/yaml/YamlPlanReaderTest.java

```diff
@@ -292,7 +293,8 @@ public void checkConversionForBuildPlan() throws YamlPlanValidationException {
 				}
 			}
 
-			JsonNode expectedYaml = yamlReader.getYamlMapper().readTree(yamlPlanFile);
+			String yamlPlanExpectedContent = replaceDynamicValuesInExpectedInput(yamlPlanFile);
```

Comment:
I have some difficulties to understand the purpose of this test and thus evaluate this change. Could you summarize it quickly or add a comment directly?